sentiment_mining.py

```diff
@@ -0,0 +1,36 @@
+import nltk
+import random
+from nltk.corpus import movie_reviews
+
+documents = [(list(movie_reviews.words(fileid)), category)
+             for category in movie_reviews.categories()
+             for fileid in movie_reviews.fileids(category)]
+
+random.shuffle(documents)
+
+all_words = []
+
+for w in movie_reviews.words():
+    all_words.append(w.lower())
```

Comment:
You could also use list comprehensions here :smile: 

```suggestion
all_words = [w.lower() for w in movie_reviews.words()]
```